core/templates/forms/layouts/advanced.html

```diff
@@ -0,0 +1,10 @@
+<div class="row">
+  <div class="row mb-3">
+    <details ontoggle="rememberAdvancedToggle(event)">
+      <summary>Advanced</summary>
+      {% for field in fieldset.fields %}
+        <div class="row">{% include "babybuddy/form_field.html" %}</div>
+      {% endfor %}
+    </details>
```

Comment:
My only objection here is that the details element feels very _not_ mobile friendly -- it's a small target link to click and it's right above the submit button. Any thoughts on how we would style to improve this?

Reply:
Agree i can add some styles to improve it, will add a screenshot when done

Reply:
<img width="881" alt="image" src="https://github.com/babybuddy/babybuddy/assets/1640136/5e23bec8-64d0-4f0e-9b16-2de44036ebe4">
<img width="707" alt="image" src="https://github.com/babybuddy/babybuddy/assets/1640136/25f2eb40-2803-4dc3-8cdb-244f2bb1c4a6">
<img width="385" alt="image" src="https://github.com/babybuddy/babybuddy/assets/1640136/003fd679-2437-4730-a60d-dcc2c63daa10">
<img width="376" alt="image" src="https://github.com/babybuddy/babybuddy/assets/1640136/2bd268f9-c96b-4ace-bd40-6beccc7e06c0">

Reply:
https://github.com/babybuddy/babybuddy/pull/760/commits/b315b1e3dce11920cd29cfc80933c5de41eeb62c